wallet/src/Pos/Wallet/Web/Swagger/Instances/Schema.hs

```diff
@@ -69,6 +70,7 @@ instance ToSchema      CT.CCoin
 instance ToSchema      CT.CInitialized
 instance ToSchema      CT.CElectronCrashReport
 instance ToSchema      CT.CUpdateInfo
+instance ToSchema      CT.NewBatchPayment
```

Comment:
@neongreen I now realise something quite disconcerting: due to the fact we are deriving `ToSchema` this way, I have the suspect (please debunk!) that this is using the `Generic` instance under the hood, which means that our Swagger API would look like crap despite we have a [decent](https://github.com/input-output-hk/cardano-sl/pull/2272/files#diff-250ee2a3dd2e1fe48fb350c1fcd92947R57) JSON instance. I did fail to appreciate this when this patch was first submitted.

In case I'm not being mental, could you actually write a proper `ToSchema` using something like [sketchSchema](https://hackage.haskell.org/package/swagger2-2.1.5/docs/src/Data.Swagger.Internal.Schema.html#sketchSchema)?

(needless to say this is all fixed in the new API, but we need to deliver exchanges a spec which actually matches the JSON input, or our Swagger spec will be of no use to them).